packages/stores/types.ts

```diff
@@ -233,6 +233,22 @@ type TTradersHubStore = {
     closeModal: () => void;
     content_flag: any;
     openModal: (modal_id: string, props?: any) => void;
+    selected_account_type: 'real' | 'demo';
+    platform_real_balance: { balance: number; currency: string };
+    cfd_demo_balance: { balance: number; currency: string };
+    platform_demo_balance: { balance: number; currency: string };
+    cfd_real_balance: { balance: number; currency: string };
```

Comment:
`{ balance: number; currency: string }`
I would suggest to move it to separate type and reuse it